src/assets/src/title.tsx

```diff
@@ -1,5 +1,20 @@
+import React from "react";
+import { Helmet } from "react-helmet-async";
+
 const BaseTitle = "Remote Office Hours Queue";
 
 export const createTitle = (title: string) => {
     return `${BaseTitle} - ${title}`;
-}
\ No newline at end of file
+}
+
+interface HelmetTitleProps {
+    title: string;
+}
+
+export const HelmetTitle: React.FC<HelmetTitleProps> = ({ title }) => {
+    return (
+        <Helmet>
+            <title>{createTitle(title)}</title>
+        </Helmet>
+    );
+};
```

Comment:
I was just adding the comment  to the file rather to the git commit. https://github.com/tl-its-umich-edu/remote-office-hours-queue/commit/8e4172f15e4b1002a62513d56ff7d647c3418cd3#r157180382